tests/test_xlora.py

```diff
@@ -128,6 +129,8 @@ def test_functional(self, tokenizer, model):
         )
         assert torch.isfinite(outputs[: inputs.shape[1] :]).all()
 
+    # TODO: remove the skip when 4.45 is released!
+    @pytest.mark.skipif(int(transformers.__version__.split(".")[1]) < 45, reason="Requires transformers >= 4.45")
```

Comment:
Let's do the version check similar to how we do it here:

https://github.com/huggingface/peft/blob/c9f7240afc616d72ea62fc9fba7bb53e858fbde1/src/peft/peft_model.py#L1670

This should be more robust. It requires the import: `import packaging.version`.